### ISSUE TITLE
Custom start / clean command or ability to hide the actions row

### ISSUE THREAD

**Issue by scottsweb** [denysdovhan/vacuum-card#91]
**Is your feature request related to a problem? Please describe.**

I have a set of input toggles used for choosing the rooms I wish to clean, these then get passed to a script and are used for custom cleaning routines. It is very rare that I do a complete 'clean'.  Some more flexibility in the actions row would be useful.

**Describe the solution you'd like**

It would be nice to have a little more control over the start/clean button or the ability to hide it.

**Describe alternatives you've considered**

Right now I will probably use a row of custom buttons instead.

**Additional context**

Custom script can be seen here: https://github.com/scottsweb/ham/blob/master/homeassistant/python_scripts/monk.py

**Comment by fi-sch** [comment]
+1 for that :)

**Comment by krystiancharubin** [comment]
You can define a custom action that will run your script. It would be segment clean just like you can do in the app.

**Comment by scottsweb** [comment]
I ended up using CSS to make something that looked like this:

![Screenshot 2020-08-25 at 14 32 35](https://user-images.githubusercontent.com/411945/91174701-eefee880-e6df-11ea-8273-904e3b20fab8.png)

The goal being to remove the default play/clean action as I always want people to be selective about the rooms that are cleaned. It works ok.

**Comment by gytisgreitai** [comment]
@scottsweb can you post the full code for newbies like me? I'm looking for something similar

**Comment by scottsweb** [comment]
@gytisgreitai you can find the UI config here: https://github.com/scottsweb/ham/blob/master/homeassistant/ui-lovelace.yaml#L543, my repo has other things you may find useful too: https://github.com/scottsweb/ham/tree/master/homeassistant

**Comment by denysdovhan** [comment]
I'll be grateful if someone could try to make a PR. Unfortunately, I don't have time to work on that feature.

**Comment by palengh** [comment]
Here is my setup, using custom scripts. I added the scripts to the custom buttons using these lines in the card config.

```
actions:
  - name: Kitchen table
    service: script.vacuum_under_kitchen_table
    icon: 'mdi:table-chair'
```

<img width="401" alt="Skjermbilde 2020-09-23 kl  11 08 13" src="https://user-images.githubusercontent.com/12332161/93991903-26b78980-fd8d-11ea-922e-030304f1fff0.png">

I have also a script that moves the vacuum to a starting point (silently moving without fan running) before starting to clean the entire house. I then make sure it starts in a spot of the house that we use the most; the kitchen, then it moves to the living room performing its standard clean all house program.

```  vacuum_under_kitchen_table:
    alias: "Vacuum under the kitchen table"
    sequence:
      - service: vacuum.send_command
        data:
          entity_id: vacuum.albertinne
          command: app_goto_target
          params: [40300,26200]
      - delay: '00:00:30'     # Wait untill starting moving (not Idle)
      - wait_template: "{{ is_state('vacuum.albertinne', 'idle') }}"
      - service: vacuum.send_command
        data:
          entity_id: vacuum.albertinne
          command: app_zoned_clean
          params: [[40300,24400,43000,26200,1]]
      - service: notify.adults
        data:
          title: chezENGH smarthouse
          message: 'Albertinne started to clean under the kitchen table at {{now().strftime("%H:%M")}}'
```
